[0.000000] (-) TimerEvent: {}
[0.004532] (pca9685_control) JobQueued: {'identifier': 'pca9685_control', 'dependencies': OrderedDict()}
[0.010239] (pca9685_control) JobStarted: {'identifier': 'pca9685_control'}
[0.098838] (-) TimerEvent: {}
[0.200474] (-) TimerEvent: {}
[0.301302] (-) TimerEvent: {}
[0.403054] (-) TimerEvent: {}
[0.506250] (-) TimerEvent: {}
[0.607104] (-) TimerEvent: {}
[0.707984] (-) TimerEvent: {}
[0.808970] (-) TimerEvent: {}
[0.912376] (-) TimerEvent: {}
[1.013292] (-) TimerEvent: {}
[1.115005] (-) TimerEvent: {}
[1.215830] (-) TimerEvent: {}
[1.316761] (-) TimerEvent: {}
[1.420269] (-) TimerEvent: {}
[1.524066] (-) TimerEvent: {}
[1.625181] (-) TimerEvent: {}
[1.725971] (-) TimerEvent: {}
[1.830488] (-) TimerEvent: {}
[1.932048] (-) TimerEvent: {}
[1.962338] (pca9685_control) StderrLine: {'line': b'Traceback (most recent call last):\n  File "/usr/lib/python3/dist-packages/colcon_core/executor/__init__.py", line 91, in __call__\n    rc = await self.task(*args, **kwargs)\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/usr/lib/python3/dist-packages/colcon_core/task/__init__.py", line 93, in __call__\n    return await task_method(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/usr/lib/python3/dist-packages/colcon_ros/task/ament_python/build.py", line 51, in build\n    setup_py_data = get_setup_data(self.context.pkg, env)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/usr/lib/python3/dist-packages/colcon_core/task/python/__init__.py", line 20, in get_setup_data\n    return dict(pkg.metadata[key](env))\n                ^^^^^^^^^^^^^^^^^^^^^^\n  File "/usr/lib/python3/dist-packages/colcon_ros/package_augmentation/ros_ament_python.py", line 57, in getter\n    return get_setup_information(\n           ^^^^^^^^^^^^^^^^^^^^^^\n  File "/usr/lib/python3/dist-packages/colcon_python_setup_py/package_identification/python_setup_py.py", line 249, in get_setup_information\n    _setup_information_cache[hashable_env] = _get_setup_information(\n                                             ^^^^^^^^^^^^^^^^^^^^^^^\n  File "/usr/lib/python3/dist-packages/colcon_python_setup_py/package_identification/python_setup_py.py", line 296, in _get_setup_information\n    result = subprocess.run(\n             ^^^^^^^^^^^^^^^\n  File "/usr/lib/python3.12/subprocess.py", line 571, in run\n    raise CalledProcessError(retcode, process.args,\nsubprocess.CalledProcessError: Command \'[\'/usr/bin/python3\', \'-c\', \'import sys;from contextlib import suppress;exec("with suppress(ImportError):    from setuptools.extern.packaging.specifiers    import SpecifierSet");exec("with suppress(ImportError):    from packaging.specifiers import SpecifierSet");from distutils.core import run_setup;dist = run_setup(    \\\'setup.py\\\', script_args=(\\\'--dry-run\\\',), stop_after=\\\'config\\\');skip_keys = (\\\'cmdclass\\\', \\\'distclass\\\', \\\'ext_modules\\\', \\\'metadata\\\');data = {    key: value for key, value in dist.__dict__.items()     if (        not key.startswith(\\\'_\\\') and         not callable(value) and         key not in skip_keys and         key not in dist.display_option_names    )};data[\\\'metadata\\\'] = {    k: v for k, v in dist.metadata.__dict__.items()     if k not in (\\\'license_files\\\', \\\'provides_extras\\\')};sys.stdout.buffer.write(repr(data).encode(\\\'utf-8\\\'))\']\' returned non-zero exit status 1.\n'}
[1.964678] (pca9685_control) JobEnded: {'identifier': 'pca9685_control', 'rc': 1}
[1.974261] (-) EventReactorShutdown: {}
